codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/SymbolVisitor.java

```diff
@@ -228,15 +235,34 @@ public Symbol memberShape(MemberShape shape) {
 
     @Override
     public Symbol timestampShape(TimestampShape shape) {
-        // TODO: add once dependency support is in
-        return createSymbolBuilder(shape, "nil").build();
+        return createSymbolBuilder(shape, "*time.Time")
+                .addReference(createNamespaceReference(GoDependency.TIME))
+                .build();
     }
 
     private Symbol.Builder createSymbolBuilder(Shape shape, String typeName) {
-        return Symbol.builder().putProperty("shape", shape).name(typeName);
```

Comment:
nit: Why the change? If you want to keep, moving `.putProperty` seems right too.

Reply:
That's an artifact of other changes I was considering, but went back on. Looks like I missed putting this back.